rmc-docs/src/tutorial-loops-unwinding.md

```diff
@@ -0,0 +1,129 @@
+# Loops, unwinding, and bounds
+
+Consider code like this:
+
+```rust
+{{#include tutorial/loops-unwinding/src/lib.rs:code}}
+```
+
+And a proof harness like this:
+
+```rust
+{{#include tutorial/loops-unwinding/src/lib.rs:rmc}}
+```
+
+When we run RMC on this, we run into an unfortunate result: non-termination.
+
+> **NOTE:** Presently, [due to a bug](https://github.com/model-checking/rmc/issues/493), this is especially bad: we don't see any output at all.
+> You are supposed to see some log lines that might give some clue that an infinite loop is occurring.
+> If RMC doesn't terminate, it's almost always the problem that this section is covering, however.
+
+To verify programs like this, we really need to do two things:
+
+1. Create an upper bound on the size of the problem.
+We've actually already done part of this: our proof harness seems to be trying to set an upper limit of 10.
+2. Tell RMC about this limit, if it's not able to figure it out on its own.
+
+> **NOTE:** In the future, RMC may eventually support specifying _loop invariants_, which allow us to do away with fixed upper bounds like this.
+> That support is not ready yet, however.
+
+Bounding proofs like this means we may no longer be proving as much as we originally hoped.
+Who's to say, if we prove everything works up to size 10, there isn't a novel bug lurking, expressible only with problems of size 11+?
+But, let's get back to the practical issue at hand.
+
+We can "make progress" in our work by giving RMC a global bound on the problem size.
```

Comment:
Can we mention the option at this point, i.e. something along the lines of: This is done via specifying `--cbmc-args --unwind <bound>`, where `<bound>` is the bound to specify.